inductiva/storage/README.md

```diff
@@ -0,0 +1,46 @@
+# Storage Management
+
+The Inductiva API provides you with the tools to manage your cloud storage effectively. Currently, we support Google Cloud, but we're working on expanding our services to include multiple cloud platforms soon!
```

Comment:
Does it make sense to mention Google Cloud storage or any other storages? We will be taking care of their effective usage I don't think it is necessary for users to know where exactly their storage is, IMO.